### DISCUSSION TITLE
Is it possible to display the thumbnail each post in mobile view?

### DISCUSSION THREAD

**Discussion by inonz987** [chipzoller/hugo-clarity#434] [Q&A] [answered]
![blog](https://user-images.githubusercontent.com/38621846/226842231-d4dbbe17-841f-433d-bd16-f0d4dbc14c32.png)

I would like the thumbnail to be displayed in both mobile and desktop modes. Your help is greatly appreciated.

**Comment by chipzoller** [comment, marked as answer]
We aren't supporting this right now but you're welcome to submit a PR.

**Reply by inonz987** [reply]
Thank you. I will try to work around it and try to submit a PR if it works.
